### PR TITLE
remove unnecessary workspace relocation script and use jenkins workspace

### PR DESCRIPTION
**Pull request by gdams**
test build: https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/build-test/job/jobs/job/jdk11u/job/jdk11u-windows-x64-hotspot/38/

### CODE REVIEW

File: pipelines/src/main/groovy/testDoubles/ContextStub.groovy [hunk 1]
```diff
@@ -64,4 +64,6 @@ class ContextStub {
     Configuration load(String s) {}
 
     String readFile(String s) {}
+
+    String ws(String s, Closure) {}
```
Comment (karianna) [inline pipelines/src/main/groovy/testDoubles/ContextStub.groovy]: ws -> workspace unless ws has special meaning :-)

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -437,6 +437,36 @@ class Build {
         return fileName
     }
 
+    def buildScripts(context, cleanWorkspace, buildConfig, filename) {
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy]: Can this function be split up?  It seems to do two disparate things.

File: pipelines/src/main/groovy/testDoubles/ContextStub.groovy (hunk 1 above)
Comment (gdams) [reply pipelines/src/main/groovy/testDoubles/ContextStub.groovy]: it has to be called ws

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 2 above)
Comment (gdams) [reply pipelines/build/common/openjdk_build_pipeline.groovy]: @karianna I didn't want to modify the function as it's existing code, happy to split up later but that should be a separate PR

**Comment by gdams** [comment]
test build: https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/build-test/job/jobs/job/jdk11u/job/jdk11u-windows-x64-hotspot/40/

**Pull request merged**